src/Controller/OcrController.php

```diff
@@ -262,6 +287,22 @@ public function apiAvailableLangsAction(): JsonResponse {
 		] );
 	}
 
+	/**
+	 * phpcs:disable MediaWiki.Commenting.FunctionAnnotations.UnrecognizedAnnotation
+	 * @Route("/api/transkribus/available_line_ids", name="apiLineIds", methods={"GET"})
+	 * OA\Response(response=200, description="List of available line detection model IDs, in JSON format")
+	 * phpcs:enable
+	 * @return JsonResponse
+	 */
+	public function apiAvailableLineDetectionModelIds(): JsonResponse {
+		$this->request->query->set( 'engine', 'transkribus' );
+		$this->setup();
+		return $this->getApiResponse( [
+			'engine' => 'transkribus',
```

Comment:
As `transkribus` is now in the API endpoint URL, there's probably no need to also list it in the returned data.